optimize/self-managed/optimize-deployment/configuration/system-configuration-platform-8.md

```diff
@@ -17,11 +17,22 @@ description: "Connection to Camunda 8."
 
 <span class="badge badge--platform">Camunda 8 Self-Managed only</span>
 
-For more information on multi-tenancy in Camunda 8 Self-Managed environments, refer to [this page](./multi-tenancy.md).
+For more information on multi-tenancy in Camunda 8 Self-Managed environments, refer
+to the [multi-tenancy documentation](./multi-tenancy.md).
 
 To use multi-tenancy, the feature must be enabled across all components.
 
 | YAML path                  | Default value | Description                                              |
 | -------------------------- | ------------- | -------------------------------------------------------- |
 | multitenancy.enabled       | false         | Enables the Camunda 8 multi-tenancy feature in Optimize. |
 | security.auth.ccsm.baseUrl | null          | The base URL of Identity.                                |
+
+### Settings related to Camunda 8 native UserTasks
```

Comment:
@PHWaechtler isn't that "Zebee user task" (Term used by tasklist)

Reply:
I dont know, this is the term you told me will be used when we initially discussed the docs. Should it be changed to `Zeebe user task`?